src/main/java/edu/hm/hafner/analysis/parser/FlawfinderParser.java

```diff
@@ -0,0 +1,52 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.RegexpLineParser;
+
+/**
+ * A parser for the flawfinder warnings.
+ *
+ * @author Dom Postorivo
+ */
+public class FlawfinderParser extends RegexpLineParser {
+    private static final long serialVersionUID = 8088991846076174837L;
+
+    private static final String FLAWFINDER_WARNING_PATTERN = "^(.*):(\\d+): .\\[([012345])\\] \\(([a-z0-9]*)\\) (.*)$";
```

Comment:
Can you please use named groups? This makes the regex and matchers better readable, see [EclipseParser.java#L28](https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/EclipseParser.java#L28) for an example

Reply:
Done. I was using the RuboCopParser as an example. If I have time I may go back and clean it up.